Lib/test/test_sqlite3/test_dbapi.py

```diff
@@ -1138,6 +1138,13 @@ def test_blob_write_error_length(self):
         with self.assertRaisesRegex(ValueError, "data longer than blob"):
             self.blob.write(b"a" * 1000)
 
+        self.blob.seek(0, SEEK_SET)
+        n = len(self.blob)
+        self.blob.write(b"a" * (n-1))
+        self.blob.write(b"a" * 1)
+        with self.assertRaisesRegex(ValueError, "data longer than blob"):
```

Comment:
Come to think of it, I don't like the sound of this exception message. I wonder if _"cannot write past end of blob"_ would be better.

Reply:
I think "data longer than blob" is fine, but happy to leave the decision up to you.